README.md

```diff
@@ -15,7 +15,7 @@ Grafana allows you to query, visualize, alert on and understand your metrics no
 - **Alerting:** Visually define alert rules for your most important metrics. Grafana will continuously evaluate and send notifications to systems like Slack, PagerDuty, VictorOps, OpsGenie.
 - **Mixed Data Sources:** Mix different data sources in the same graph! You can specify a data source on a per-query basis. This works for even custom datasources.
 
-## Get started
+## Get Started
```

Comment:
we do not use title case

Reply:
Correct, we use sentence case. 

Please make the following change:
"Mixed data sources" and "Get started".

Reply:
Done

Reply:
@Rowayda-Khayri was this fixed? Thanks!

Reply:
> @Rowayda-Khayri was this fixed? Thanks!

"Get Started" was fixed. But "Mixed data sources" wasn't changed in my PR!

Reply:
OK - would you want to fix it now or should we do a new PR? Let me know :)

Reply:
@achatterjee-grafana OK, I'll make a new PR for it